core/templates/dev/head/galleries/gallery.html

```diff
@@ -2,11 +2,11 @@
 
 <!-- Gallery page is a special case: we switch the main and sub titles -->
 {% block maintitle %}
-  Gallery
+  <['I18N_GALLERY_PAGE_TITLE' | translate]>
```

Comment:
We use `<span translate>` in base.html, and a filter here. I'd prefer to use a consistent style throughout -- probably the former.